app/controllers/kaui/invoices_controller.rb

```diff
@@ -7,7 +7,7 @@ def index
 
       @ordering = params[:ordering] || (@search_query.blank? ? 'desc' : 'asc')
       @offset = params[:offset] || 0
-      @limit = params[:limit] || 50
```

Comment:
I think we should keep the default limit to 50 (it applies to both the per tenant and per account screens).